### PR TITLE
fixes issue #1208:remove references to mongodb

### PR DESCRIPTION
**Pull request by prangya244**
removed references to Mongo DB

### CODE REVIEW

File: data-structures/4-semesters-of-computer-science.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ If you are in the San Francisco Bay Area and you get a SFPL library card, you ge
 ### Specific Things to Learn
 - Things about the things
 - More things about the things
-	- This is a sub-thing about the things
```
Comment (vegetabill) [inline data-structures/4-semesters-of-computer-science.md]: Please remove any changes not related to this PR.

File: databases/relational-databases.md [hunk 2]
```diff
@@ -214,8 +214,6 @@ The table containing the foreign key is called the child table, and the table co
        Strings are the simplest data type in Redis and are simple key/value entries.
   - graph [(Neo4j)](https://github.com/JNOSQL/artemis-demo/tree/master/artemis-demo-java-se/graph-neo4j)
     - Neo4j is a graph database management system developed by Neo4j, Inc. Described by its developers as an ACID-compliant transactional database with native graph storage and processing. 
-  - document [(mongo)](https://docs.mongodb.com/getting-started/cpp/documents/)
-    - MongoDB is a document database: each record in a MongoDB collection is document. Documents are a structure composed of file and value pairs,         similar to JSON objects or other mapping data types.
```
Comment (vegetabill) [inline databases/relational-databases.md]: Please keep these. Sorry for forgetting to include them on the ignore list.

File: README.md [hunk 3]
```diff
@@ -218,8 +218,6 @@ These are not blockers for any of the core sequence and so they are worked into
 - [Datetime](/datetime/datetime.md)
 - [Data Visualization](/data-visualization/data-visualization.md)
 - [Adding Chai/Mocha Tests to your project](/testing-and-tdd/mocha-testing.md)
-- [MongoDB](/databases/mongo-db.md)
-- [Mongoose](/databases/mongoose.md)
```
Comment (vegetabill) [inline README.md]: These are still electives so please keep these lines

File: express-js/server-side-project.md [hunk 4]
```diff
@@ -45,9 +45,9 @@
 - Explain the MVC framework.
 - Create a basic web app using Node.js for the server and Express.js as the MVC framework.
 
-## Mongo
-- Set up a MongoDB database.
-- Create databases and collections in MongoDB.
+## PostgreSQL
+- Set up a Postgres database.
+- Create databases and collections in Postgres.
```
Comment (vegetabill) [inline express-js/server-side-project.md]: change to "Create tables." (collections is a Mongo concept)

File: objectives/9.md [hunk 5]
```diff
@@ -12,11 +12,11 @@ Hook up user actions to state changes in the UI.
...
 
-## [Assessment Outline 9.1: Mongo](projects/mongo-assessment.md)
+## [Assessment Outline 9.1: Mongo](projects/postgres-assessment.md)
 
-### MongoDB
-* Set up MongoDB on their laptops
-* Create databases and collections in MongoDB
+### PostgreSQL
+* Set up PostgreSQL on their laptops
+* Create databases and collections in PostgreSQL
```
Comment (vegetabill) [inline objectives/9.md]: Create tables

File: projects/postgres-assessment.md [hunk 6]
```diff
@@ -1,17 +1,17 @@
-## Assessment #9.1: A Quick MongoDB Challenge
+## Assessment #9.1: A Quick PostgreSQL Challenge
```
Comment (vegetabill) [inline projects/postgres-assessment.md]: This file was on the list of ignored files in the issue. Please undo these changes.

File: projects/kickball-signups-project.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 
 ### Languages
 - JS
-- MongoDB
+- PostgreSQL
```
Comment (vegetabill) [inline projects/kickball-signups-project.md]: This file was mentioned as to be ignored. Please undo

**Comment by prangya244** [comment]
@vegetabill  I have done the required changes that you asked for. Kindly review them and let me know if any more changes is required.

File: data-structures/4-semesters-of-computer-science.md (hunk 1 above)
Comment (alodahl) [reply data-structures/4-semesters-of-computer-science.md]: This file should not be here at all. This may help: https://stackoverflow.com/questions/1817766/how-to-revert-to-origins-master-branchs-version-of-file

Comment (alodahl) [reply data-structures/4-semesters-of-computer-science.md]: Welcome to the world of git! You can do it 😉

File: objectives/9.md [hunk 8]
```diff
@@ -12,11 +12,11 @@ Hook up user actions to state changes in the UI.
 * Add through testing with Enzyme and Jest to one of your React components
 _______________
 
-## [Assessment Outline 9.1: Mongo](projects/mongo-assessment.md)
+## [Assessment Outline 9.1: Mongo](projects/postgres-assessment.md)
```
Comment (alodahl) [inline objectives/9.md]: is this a real URL? I get a 404.

**Comment by alodahl** [comment]
@prangya244 Try your hand at resolving this conflict, too.  
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github

File: databases/installing-postgresql.md [hunk 9]
```diff
@@ -14,6 +14,8 @@
 
 ### Install PostgreSQL on Mac
 
+
+- Follow the [Postgres Homebrew Instructins](https://wiki.postgresql.org/wiki/Homebrew)
```
Comment (alodahl) [inline databases/installing-postgresql.md]:
```suggestion
- Follow the [Postgres Homebrew Instructions](https://wiki.postgresql.org/wiki/Homebrew)
```

**Comment by Kundan28** [comment]
Updated in score sheet too.
